test/controllers/api/v1/items_controller_test.rb

```diff
@@ -49,4 +61,12 @@ def setup
     assert_response :success
     assert_equal merchant.id, json_response["id"]
   end
+
+  test "#best_day" do
+    get :best_day, format: :json, id: item.id
+
+    assert_response :success
+    response = {"best_day"=>"2012-03-27T14:54:09.000Z"}
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.